core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java

```diff
@@ -61,7 +61,10 @@
   public PollingMasterInquireClient(List<InetSocketAddress> masterAddresses,
       AlluxioConfiguration alluxioConf,
       UserState userState) {
-    this(masterAddresses, () -> new ExponentialBackoffRetry(20, 2000, 30),
+    this(masterAddresses, () -> RetryUtils.defaultClientRetry(
+        alluxioConf.getDuration(PropertyKey.USER_RPC_RETRY_MAX_DURATION),
```

Comment:
How does changing this affect other tests? Will it still behave similarly to the previous retry policy?

Reply:
The old way has an exponential backoff max of 2s * 30 total attempts. It takes 7 retries before hitting the 2s cap. This results in a max wait time of 46s + ~3s = ~50s. The values from PK defaults result in a max of 2min before stopping.

This is the profile for time between retries for each

![image](https://user-images.githubusercontent.com/3464508/73977610-8422c700-48df-11ea-9f44-2dfaa8ee58db.png)

This is a graph for total time spent vs # of retries

![image](https://user-images.githubusercontent.com/3464508/73977880-03b09600-48e0-11ea-8747-efababc18b1d.png)

The wait time maxes out much higher for the polling client when using the default RPC retries

Equations:
![image](https://user-images.githubusercontent.com/3464508/73978010-4c684f00-48e0-11ea-87a3-1651c524f170.png)


Most cases there won't be a difference. The worst thing that will happen is that it results in users needing to wait longer for the polling client to determine who the leader master is. I can add more PKs so that the retry profile for the polling client match more closely with the current values than using the generic RPC retry parameters.

@ggezer I requested your review for the reason stated above. Do you think there are other repercussions to changing the profile? Should we try to keep the values what they are now?

Reply:
As discussed offline, fail-over scenario is not effected as the tight-loop of finding a responding master address won't be done under retry policy. Quick swipe on code shows no usage of inquire clients outside of RPC context, so I think It's fine to reuse these parameters.